### PR TITLE
doc: README + initial documentation

### PR DESCRIPTION
**Pull request by sh0rez**
Initial documentation work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Overview
+
+Tanka is a configuration utility for [Kubernetes](https://kubernetes.io/). It
+leverages [Jsonnet](https://jsonnet.org) to allow highly dynamic configuration.
+
+- **:repeat: `ksonnet` drop-in replacement**: Tanka aims to provide the same
+  workflow as `ksonnet`: `show`, `diff` and `apply` are just where you expect
+  them.
+- **:nut_and_bolt: integrates with the ecosystem**: Tanka does re-invent the
```
Comment (rfratto) [inline README.md]: s/does/doesn't/ ?

Comment (sh0rez) [reply README.md]: oops

File: docs/environments.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+same application available in multiple regional zones / datacenters.
+
+An environment does not need to be created, it rather just exists as soon as a
+main.jsonnet is found somewhere in the tree of a `rootDir`. This also means that
+an environment by definition is equivalent to a [`baseDir`](directory-structure.md#base-directory-basedir).
+
+## Configuration
+To correctly deal with an environment, Tanka needs some additional information
+about it.
+
```
Comment (malcolmholmes) [inline docs/environments.md]: Presumably this additional info is expected in `spec.json` next to `main.jsonnet`? It doesn't says so.

Comment (sh0rez) [reply docs/environments.md]: Oh yeah, missed to add that info

**Comment by sh0rez** [comment]
This is ready for review!

Site now builds using `mkdocs`, hosted on `tanka.dev` using netlify. See the status check `deploy/netlify` for a preview.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+![Tanka Banner](docs/banner.png)
+
+[![Build Status](https://cloud.drone.io/api/badges/grafana/tanka/status.svg)](https://cloud.drone.io/grafana/tanka)
+![Golang](https://img.shields.io/badge/language-Go-blue)
+![GitHub contributors](https://img.shields.io/github/contributors/grafana/tanka)
+![GitHub release](https://img.shields.io/github/release/grafana/tanka)
+![License](https://img.shields.io/github/license/grafana/tanka)
+
+Tanka is a configuration utility for [Kubernetes](https://kubernetes.io/). It
+leverages [Jsonnet](https://jsonnet.org) to allow highly dynamic configuration.
```
Comment (tomwilkie) [inline README.md]:
```suggestion
Tanka is part a configuration management system for [Kubernetes](https://kubernetes.io/). 
Tanka uses the [Jsonnet](https://jsonnet.org) language to allow flexible, reusable and concise configuration.
```

Comment (tomwilkie) [reply README.md]: (Looks like the github suggestion didn't quite work, but you get the idea.)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  workflow as `ksonnet`: `show`, `diff` and `apply` are just where you expect
+  them.
+- **:nut_and_bolt: integrates with the ecosystem**: Tanka doesn't re-invent the
+  wheel. It rather makes heavy use of what is already there:
+  [`jsonnet-bundler`](https://github.com/jsonnet-bundler/jsonnet-bundler) for
+  package management and
+  [`kubectl`](https://kubernetes.io/docs/reference/kubectl/overview/) for
+  communicating with Kubernetes clusters.
+- **:hammer: powerful:** Being a `jsonnet`-compatibility layer for Kubernetes,
+  it removes the limitations of static configuration languages.
```
Comment (tomwilkie) [inline README.md]:
```suggestion
  it removes the limitations of static (or template-based) configuration languages.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+- **:rocket: used in production**: We already use Tanka internally at
```
Comment (tomwilkie) [inline README.md]:
```suggestion
- **:rocket: used in production**: We already use Tanka internally at
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+  Grafana for all of our Kubernetes configuration needs.
```
Comment (tomwilkie) [inline README.md]:
```suggestion
  Grafana Labs for all of our Kubernetes configuration needs.
```

**Comment by tomwilkie** [comment]
Can you put the Grafana Labs logo on the front page too please?

**Comment by tomwilkie** [comment]
Once more thought: you could include a like to jsonnet-libs and mixins to show examples of "tanka-compatible" libraries.

**Pull request merged**